lib/components/term.js

```diff
@@ -98,9 +106,24 @@ export default class Term extends Component {
 
     this.getScreenNode().addEventListener('mouseup', this.handleMouseUp);
     this.getScreenNode().addEventListener('mousedown', this.handleMouseDown);
+    this.getScreenNode().addEventListener('contextmenu', this.handleRightClick);
     terms[this.props.uid] = this;
   }
 
+  handleRightClick(e) {
+    e.preventDefault();
+    e.stopPropagation();
+
+    const {quickEdit} = config.getConfig();
+    const show = !quickEdit;
+
+    const contextMenu = Menu.buildFromTemplate(contextMenuTemplate);
+    if (show) {
+      // throttle popup so right click has time to highlight text
+      setTimeout(() => contextMenu.popup(getCurrentWindow()), 100);
```

Comment:
Do not hesitate to ask question in this PR or directly on slack!
Really happy to help you on this awesome feature!